services/app-web/src/pages/StateSubmission/ContractDetails/ContractDetailsSchema.ts

```diff
@@ -132,5 +134,8 @@ export const ContractDetailsFormSchema = (
         modifiedNonRiskPaymentArrangements: yesNoError(
             'modifiedNonRiskPaymentArrangements'
         ),
+        statutoryRegulatoryAttestation: activeFeatureFlags['438-attestation']
```

Comment:
If you move this key up in the object to be first in the list does it switch the error summary order? Where its located in the list of errors right now is confusing. If moving it up fixes things, please also add a comment to this file clarifying that order of keys matters in this file. 
![Screenshot 2023-11-14 at 2 00 50 PM](https://github.com/Enterprise-CMCS/managed-care-review/assets/10750442/72985f1d-d560-4a22-bca0-77480187acea)